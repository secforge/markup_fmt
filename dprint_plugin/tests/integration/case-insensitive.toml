[vue_custom_block]
default = "lang-attribute"
i18n = "lang-attribute"
docs = "none"
