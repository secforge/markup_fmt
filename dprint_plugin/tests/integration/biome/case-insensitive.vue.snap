---
source: dprint_plugin/tests/integration.rs
assertion_line: 218
---
<template>
  <div>Test</div>
</template>

<I18N lang="json">
{
	"en": {
		"hello": "Hello",
		"goodbye": "Goodbye",
		"welcome": "Welcome to our application"
	},
	"de": {
		"hello": "Hallo",
		"goodbye": "Auf Wiedersehen",
		"welcome": "Willkommen in unserer Anwendung"
	}
}
</I18N>

<DOCS>
  <p>Some    documentation    with    extra    spaces</p>
</DOCS>
